node/Cargo.toml

```diff
@@ -25,9 +25,9 @@ jsonrpc-core = '15.1.0'
 serde_json = '1.0'
 structopt = '0.3.8'
 # TODO: update parity-scale-codec to 2.2.0
-codec = { package = 'parity-scale-codec', version = '2.0.0' }
+codec = { package = 'parity-scale-codec', version = '2.3.0' }
 log = "0.4"
-serde = { version = "1.0", features = ["derive"]}
+serde = { version = "1.0.119", features = ["derive"] }
```

Comment:
nit: There is absolutely no need in such change, Cargo ignores patch versions anyway https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#caret-requirements